### PR TITLE
feat: response style (creative, precise)

### PR DESCRIPTION
**Pull request by mdjastrzebski**
### Summary
Adds options for creative and precise responses from AI

### Test plan
Hard to figure out... but with precise mode responses should not diverge too much from the initial one.

### CODE REVIEW

File: src/commands/prompt/index.ts [hunk 1]
```diff
@@ -112,10 +129,20 @@ async function runInternal(initialPrompt: string, options: PromptOptions) {
...
 
+  let style = {};
+
+  if (options.creative) {
+    style = RESPONSE_STYLE_CREATIVE;
+  }
+  if (options.precise) {
+    style = RESPONSE_STYLE_PRECISE;
+  }
+
```
Comment (mdjastrzebski) [inline src/commands/prompt/index.ts:146]: Handle case when user sets both, let's warn them in such case.

**Review by mdjastrzebski — COMMENTED** [review]
@Q1w1N almost there. check  a single comment i've left

File: src/commands/prompt/index.ts (hunk 1 above)
Comment (Q1w1N) [reply src/commands/prompt/index.ts:146]: Done

**Comment by mdjastrzebski** [comment]
Approved!

**Review by Q1w1N — APPROVED** [review]

**Pull request merged**